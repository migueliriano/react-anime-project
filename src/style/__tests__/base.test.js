import React from 'react';
import renderer from 'react-test-renderer';

import {
  Container,
  Wrapper,
  Header,
} from '../base';


describe('base style components', () => {
  it('Should render a Container Component', () => {
    const tree = renderer.create(<Container />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a Wrapper Component', () => {
    const tree = renderer.create(<Wrapper />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a Header Component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
