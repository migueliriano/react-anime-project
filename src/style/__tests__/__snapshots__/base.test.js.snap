// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base style components Should render a Container Component 1`] = `
.c0 {
  max-width: 1020px;
}

<div
  className="c0"
/>
`;

exports[`base style components Should render a Header Component 1`] = `
.c0 {
  text-align: center;
  padding-top: 30px;
}

<header
  className="c0"
/>
`;

exports[`base style components Should render a Wrapper Component 1`] = `
.c0 {
  padding: 0;
  margin: 0;
}

<section
  className="c0"
/>
`;
